lib/helpers/tasks.rb

```diff
@@ -64,6 +64,8 @@ def generate_types(environment)
   def notification(message)
     return unless settings.chatops || settings.slack_webhook
 
+    require 'plugins/chatops' if settings.chatops
```

Comment:
Don't you need this always? Just below you use the class `PuppetWebhook::Chatops` and I think this will fail if you use the slack webhook.

Reply:
Another good catch. The first line of the method will handle if the setting is enabled or not.